src/renderer/html_handlebars/helpers/theme.rs

```diff
@@ -0,0 +1,30 @@
+use handlebars::{Context, Handlebars, Helper, Output, RenderContext, RenderError};
+
+pub fn theme_option(
```

Comment:
I like the thinking behind using a `theme_option` helper append " (default)", it's much nicer than using javascript to add it after the fact :+1: